lib/train/plugins/base_connection.rb

```diff
@@ -118,10 +118,9 @@ def platform
     # This command accepts an optional data handler block. When provided,
     # inbound data will be published vi `data_handler.call(data)`. This can allow
     # callers to receive and render updates from remote command execution.
-    def run_command(cmd, &data_handler)
-      return run_command_via_connection(cmd, &data_handler) unless cache_enabled?(:command)
-
-      @cache[:command][cmd] ||= run_command_via_connection(cmd, &data_handler)
+    def run_command(cmd, override_opts = {}, &data_handler)
```

Comment:
these should be `**`'d and probably should just be `**opts` instead of `override_opts = {}`.